pkg/obfuscate/go.mod

```diff
@@ -6,6 +6,6 @@ require (
 	github.com/DataDog/datadog-go v4.8.2+incompatible
 	github.com/Microsoft/go-winio v0.5.1 // indirect
 	github.com/dgraph-io/ristretto v0.1.0
-	github.com/mailru/easyjson v0.7.7
+	github.com/mailru/easyjson v0.0.0-20170227080219-44f6bc771e3c
```

Comment:
Why are we not using a proper version? Why this pseudo-version?

Reply:
This matches the internal project we're targeting.
Unfortunately, the commit precede by two years the earlier published version (v0.7.0, September 10th 2019).
Between the complexity of an upgrade on the other side and the limited scope of the downgrade here, it felt like this was the quickest and least risky change.

Reply:
I updated to v0.7.0 and it doesn't appear to break anything.
I agree it's much cleaner than tagging a random commit from 2017 :)